test/cli_command_test/test_attach.py

```diff
@@ -182,3 +182,8 @@ def test_pool_file_invalid(self):
             self.assertEqual(e.code, os.EX_DATAERR)
         else:
             self.fail("No Exception Raised")
+
+
+# Test Attach and Subscribe are the same
+class TestSubscribeCommand(TestAttachCommand):
```

Comment:
This is unneeded as well, if we've removed the Subscribe command (same for Unsubscribe below)

Reply:
True!  :)